src/components/Error.tsx

```
@@ -0,0 +1,12 @@
+export default function Error({ message }: { message: string }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)